ansible-101/site-digitalocean.yml

```diff
@@ -19,12 +19,9 @@
   vars:
     oauth_token: "{{ lookup('ansible.builtin.env', 'DO_API_TOKEN') }}"
     ssh_pubkeys:
-      - rpolli-ed25519
-      - esavo-rsa
-      - dabbasciano-rsa
+      - d8o-mac
     droplet_names:
-      - deleteme-1
-      - deleteme-2
+      - ansible-test
 
```

Comment:
@DomenicoCutrupi don't change these lines, since these are aligned with deleteme-digitalocean.yml. Clearly you can change them on your PC.